bundle/src/main/java/com/adobe/acs/commons/fam/impl/ActionManagerImpl.java

```diff
@@ -48,14 +50,14 @@
 /**
  * Manages a pool of reusable resource resolvers and injects them into tasks
  */
-class ActionManagerImpl implements ActionManager {
+class ActionManagerImpl extends CancelHandler implements ActionManager {
 
-    private static final Logger LOG = LoggerFactory.getLogger(ActionManagerImpl.class);
+    transient private static final Logger LOG = LoggerFactory.getLogger(ActionManagerImpl.class);
```

Comment:
So that GSON doesn't serialize it in the mcp rest servlet.  The way it works is that GSON serializes a bunch of stuff and serves it up to the front-end so that it can do live status reporting, etc.